frontend/src/lib/Buttons.ts

```diff
@@ -80,11 +80,12 @@ export default class Buttons {
   }
 
   public delete(getSelectedIds: () => string[], resourceName: 'pipeline' | 'recurring run config',
-    callback: (selectedIds: string[], success: boolean) => void, useCurrentResource: boolean): ToolbarActionConfig {
+    callback: (selectedIds: string[], success: boolean) => void, useCurrentResource: boolean,
+    refreshOnComplete: boolean): ToolbarActionConfig {
```

Comment:
Isn't this always desired? If it is, we can just always call it here.
If it isn't, why can't refresh be optionally called by the callsite as part of the callback?

Reply:
It isn't desirable when you're on a specific resource's page (e.g. Run Details), and you delete that resource. In that case, a request is sent to the server to reload the page for a resource that no longer exists before redirecting to whatever page you actually want to be on.

The second part/suggestion would work. My thinking was just that this would prevent duplicate code in the callbacks

Reply:
Why not use the `useCurrentResource` flag then? Seems like it has the same information.

Reply:
No strong opinion here. I thought maybe it would be useful to have an explicit flag so that the implications are clear, but there's no reason AFAICT that we couldn't use the `useCurrentResource` (or situation where that wouldn't make sense at the moment).

I can change it if you think it's better to not add the extra flag

Reply:
I think it's a generally good idea to simplify the interface provided the caller doesn't need to know this flag is used for two different things. It seems internal to the method's implementation.

Reply:
Sounds good. It's been removed